### ISSUE TITLE
Add fastjet to conda-forge

### ISSUE THREAD

**Issue by jmduarte** [scikit-hep/fastjet#133]
Inspired by #91.

It would be great to get fastjet on conda-forge (especially in time for the first real release 3.4.0.0, which now appears to be imminent). I'm unsure if we want to make this a requirement for the release. What do others think?

@lgray @jpivarski @chrispap95 @henryiii

**Comment by lgray** [comment]
In my experience first you make a pypi release then you stick it into conda forge which follows pypi.
You can see from https://github.com/conda-forge/staged-recipes/pull/15961 a while ago that going in with an RC or something that's a little wonky can be quite some pain.

**Comment by jpivarski** [comment]
Getting fastjet into conda-forge would be great!

As @lgray said, the conda-forge is a more frictionless process with non-RC releases. Only the first three digits of the version number (3.4.0) are constrained to match FastJet's; the fourth digit is free to increase as needed, so it is not necessary to get the conda-forge deployment right on the first non-RC attempt.

Also, the way that the conda-forge package gets populated is through the PyPI source wheel (e.g. fastjet-3.4.0.0.tar.gz). As long as the environment on conda-forge can take this tarball and compile all of its packages, it should work.

I'll also bring @chrisburr into this discussion, since he's a conda expert.

**Comment by jmduarte** [comment]
In progress by @lgray here: https://github.com/conda-forge/staged-recipes/pull/21052

**Comment by jmduarte** [comment]
@lgray perhaps the blocker from before (https://github.com/conda-forge/staged-recipes/pull/21052#issuecomment-1307593991) is now resolved by #159 ?

**Comment by eduardo-rodrigues** [comment]
This is in fact done since a while, see https://github.com/conda-forge/fastjet-feedstock.

**Comment by lgray** [comment]
Sorta - we got the recipe to work once for one version, and then the next version broke again. 🎉

**Comment by matthewfeickert** [comment]
Yeah, this should maybe be left open as a reminder that we still need to get the latest versions building and also make them multiplatform (still only Linux at this point).

**Comment by eduardo-rodrigues** [comment]
Indeed. Sorry for missing those facts.

**Comment by matthewfeickert** [comment]
Nothing to be sorry about! This is a good reminder that we need to follow up on getting conda-forge setup properly across multiplatforms.

**Comment by matthewfeickert** [comment]
For a broader community project I'm working on it is going to be necessary to have working distributions of `fastjet` (including the FastJet C++ libraries) on conda-forge. It will also be necessary to have things like Issue https://github.com/scikit-hep/fastjet/issues/308 resolved.

Should `fastjet` get fixed to support the FastJet C++ libraries? Or should `fastjet-cxx` be created on conda-forge to support the C++ project as its own distribution and then have `fastjet` not worry about making the C++ available in a useful form?

**Comment by lgray** [comment]
closed by #342 and https://github.com/conda-forge/fastjet-feedstock/pull/23
